src/android/HealthPlugin.java

```diff
@@ -577,7 +588,8 @@ public void onConnectionFailed(ConnectionResult result) {
                 }
         );
         mClient = builder.build();
-        mClient.connect();
+        mClient.clearDefaultAccountAndReconnect();
```

Comment:
this will force the app to request authorisation at each requestAuthorization, which is quite annoying. Ideally, once the app is authorised, it shouldn't ask for credentials any more.